Formula/kat.rb

```diff
@@ -14,31 +14,53 @@ class Kat < Formula
 
   head do
     url "https://github.com/TGAC/KAT.git"
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
```

Comment:
If head no longer depends on `autoconf` and `automake`, you can replaced the `head do` block with the simpler:
```ruby
head "https://github.com/TGAC/KAT.git"
```

Reply:
will do